test.py

```diff
@@ -105,8 +101,5 @@ def test_add_op_4():
 
 def test_initialization():
 
-    memo = amino.Memoizer(50, 0.02, "epanechnikov")
-    memo.initialize_distances(ops)
-    
-    assert equal(memo._distance_kernel(ops[0], ops[1]), 0.5353485087893664)
+    assert equal(memo._distance_kernel(0, 1), 0.5353485087893664)
```

Comment:
Minor thing I noticed: since we reuse this number a few times it may be good to define it as a constant. It's obvious enough that it's the expected distance though so not strictly necessary.